recipes/llvm-openmp-mutex/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "llvm-openmp-mutex" %}
```

```suggestion
{% set name = "_openmp_mutex" %}
```
so that the feedstock will be named `_openmp_mutex-feedstock`